viaacode/hetarchief-client
# feature(ARC-1303): Geen toegang pagina

<img width="1705" alt="image" src="https://user-images.githubusercontent.com/113892598/233649231-a8b7c77d-9244-4cd0-856e-7513470f88e6.png">


Ticket: https://meemoo.atlassian.net/browse/ARC-1303
Proxy PR: https://github.com/viaacode/hetarchief-proxy/pull/392

To test: 

1. Login as meemoo admin
2. Search "judo" -> video is found and can be watched (first result)
3. Login as 'bezoeker'
4. Search "judo" -> video not found
5. Go to the url of the "judo" video: 
6. http://localhost:3200/zoeken/vrt/31b120ee84c54e6ba1447d3e2cc3249cef1cebe94558438085d19cfe1a771fb27fc693c4b8eb40a0998bc4869012145a
7. Error page is shown

Other item that has no access (metadata is available for meemoo admin, so error page should not be shown for meemoo admin): http://localhost:3200/zoeken/vrt/b25a2e41a32d4f3d9f85e0f47b38f01af242e66797004faa9f65d8f133f5cfdedb6ac48a338843de8da41d0473380cfc
